README.md

```diff
@@ -22,7 +22,7 @@ Not directly (after all, this is **Sprite**Glow), but it's possible if the compo
 
 ![](https://i.gyazo.com/6c92f315d8a25600bf4ec930c5b7de3e.png)
 
-### Why the glow doesn't appear?
+### Why doesn't my sprite glow?
```

Comment:
I think the original is more fitting in this case.

Reply:
How about "Why doesn't the glow appear?"